architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -432,4 +432,39 @@ to run:
 `oc delete istag/ruby:latest`
 ====
 
+[[insecure-registries]]
 
+=== Importing images from insecure registries
+An image stream can be configured to import tag and image metadata from an image
+repository that is signed with a self-signed certificate. To do that, add
```

Comment:
Not just self-signed certs. This will also make it work with plain http as well.